pkg/apis/app/v1alpha1/zz_generated.openapi.go

```diff
@@ -504,8 +504,10 @@ func schema_pkg_apis_app_v1alpha1_KogitoAppBuildObject(ref common.ReferenceCallb
 					"webhooks": {
 						VendorExtensible: spec.VendorExtensible{
 							Extensions: spec.Extensions{
-								"x-kubernetes-list-map-keys": "type",
-								"x-kubernetes-list-type":     "map",
+								"x-kubernetes-list-map-keys": []interface{}{
```

Comment:
This file shouldn't be updated.

Reply:
I didn't upate it. it was auto generated.

Reply:
Something happens in your env/dependencies:
https://github.com/kiegroup/kogito-cloud-operator/blob/master/pkg/apis/app/v1alpha1/zz_generated.openapi.go#L504-L509